generators/app/tasks/bitriseInitialization.js

```diff
@@ -10,98 +12,40 @@ module.exports = function bitriseInitialization() {
     context: this.options
   }).then(({ spinner }) => {
     spinner.stop();
-    return this.prompt([
```

Comment:
I'd keep all these prompt questions as a validation if there doesn't exist the file or if the key prop doesn't exist in the file config file. What u think? We can move them to `bitrise` or `bitriseUtils` files

Reply:
I don't know. I think it's not necessary. It's a lot of unnecessary code.

Reply:
I'm agree with Wally here.
If you move this code to another file we can create a command like `run.sh` to run the Bitrise script without clone the repo like we do here with the bootstrap: `bash <(curl -s https://raw.githubusercontent.com/Wolox/wolmo-bootstrap-react-native/master/run.sh)`.
If you delete these prompts you are forcing the user to clone the repo and fill the config file and not all users want this.
So, we can keep the config file if the user clone the repo and use the prompt questions if the user want to use the command above.
What do you think?

Reply:
Okey, I've finally did it. Please check